content/faq.md

```diff
@@ -235,39 +352,56 @@ There are two kinds of release schemes for locked up assets: linear and conditio
 
-If your lockup involved a galaxy, all of its stars will be locked up, but you will have immediate, lock-free control of the galaxy. You will likely need it to use that star.
+If your lockup involved a [galaxy](@/docs/glossary/galaxy.md), all of its [stars](@/docs/glossary/star.md) will be locked up, but you will have immediate, lock-free control of the galaxy. You will likely need it to use that star.
 
 Note that the "releasing" of stars just means that they become available for you to claim. They don't automatically get transferred to you; you have to withdraw them from the appropriate lockup contract.
 
 Linear release is the simplest: your stars will be released linearly over a period of time. Most often this is a period of four years. If you have four stars in lockup, that means you will be able to withdraw one star per year. In many cases, there is also an initial windup period which has to pass before linear releasing begins, typically one year. Since Azimuth launched in January 2019, the linear release will begin in January 2020.
 
-Conditional release is a bit more complicated. If your stars are in conditional lockup, they're likely divided over three so-called tranches. Each of these unlocks only after a unique condition is met. Since it's difficult to verify things about the real world using smart contracts, the Galactic Senate verifies that they've been met. Once the Senate marks a condition for a tranche as cleared, it starts releasing linearly over the period of a year.
+Conditional release is a bit more complicated. If your stars are in conditional
+lockup, they're likely divided over three so-called tranches. Each of these
+unlocks only after a unique condition is met. Since it's difficult to verify
+things about the real world using smart contracts, the [Galactic
+Senate](@/docs/glossary/senate.md) verifies that they've been met. Once the
+Senate marks a condition for a tranche as cleared, it starts releasing linearly
+over the period of a year.
 
 ## Urbit OS
 
 ### What is Urbit OS? {#what-is-arvo}
 
-Urbit OS, also called Arvo, is our functional and deterministic software stack. Arvo is written in Hoon, our purely functional programming language, which compiles to Nock, a compact bytecode language.
+Urbit OS, also called [Arvo](@/docs/glossary/arvo.md), is our functional and
+deterministic software stack. Arvo is written in [Hoon](@/docs/glossary/hoon.md), our purely functional
+programming language, which compiles to [Nock](@/docs/glossary/nock.md), a compact bytecode language.
 
-Instances of Urbit OS / Arvo, called 'ships', communicate as peers on what we refer to as “the Urbit OS network.” Using Urbit ID, Urbit OS ships can prove their identities to one another.
+Instances of Urbit OS / Arvo, called '[ships](@/docs/glossary/ship.md)',
```

Comment:
Either Urbit OS or Arvo, having both is awkward.

Reply:
https://github.com/urbit/urbit.org/pull/538/commits/d08381ccc8c13d6728b6720c79584d2bd0b209fb